event_seats_per_registration/models.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U.
+
+from openerp import _, api, fields, models
+from . import exceptions
+
+
+class Event(models.Model):
+    """Events enhaced with participants per registration limits."""
+
+    _inherit = "event.event"
+
+    seats_per_registration_max = fields.Integer(
```

Comment:
OK, I see there's another `seats_max` field, so it can confussing, but you can let it as `max_registration_seats` (at least you save 4 characters)